src/chrome/content/rules/US-government.xml

```diff
@@ -190,7 +213,7 @@ Fetch error: http://wtr.nhtsa.gov/ => https://wtr.nhtsa.gov/: (7, 'Failed to con
 
 	<securecookie host="^questions\.cms\.gov$" name=".*"/>
 	<securecookie host="^safeharbor\.export\.gov$" name=".*"/>
-	<securecookie host="^(?:.*\.)?(?:census|medicare|usajobs)\.gov$" name=".*"/>
+	<securecookie host="^(?:.*\.)?(?:census|(my)?medicare|usajobs)\.gov$" name=".*"/>
```

Comment:
Please simplify this to `medicare|mymedicare`, which is easier to read (and search for).